src/ontology/uberon-edit.obo

```diff
@@ -224613,6 +224613,17 @@ relationship: overlaps UBERON:0018227 ! pulmonary lymphatic vessel
 relationship: surrounds UBERON:0010369 ! secondary pulmonary lobule
 property_value: dcterms-date "2023-10-23T08:52:12Z" xsd:dateTime
 
+[Term]
+id: UBERON:8600024
+name: pulmonary venule
+def: "A venule that is part of a lung and drains oxygenated blood from pulmonary capillaries. A pulmonary venule merges into a pulmonary vein branch that run in the pulmonary interlobular septa." [PMID:22283705, PMID:30247823]
```

Comment:
```suggestion
def: "A venule that is part of a lung and drains oxygenated blood from pulmonary capillaries. A pulmonary venule merges into a pulmonary vein branch that runs in the pulmonary interlobular septa." [PMID:22283705, PMID:30247823]
```

This sounds like a "pulmonary vein branch", not a pulmonary venule, passes through a pulmonary interlobular septum. Is that aligned with the comment [here](https://github.com/obophenotype/uberon/pull/3095#discussion_r1369754009)?

Reply:
You are correct, I have to edit interlobular septa accordingly.

Reply:
A new ticket #3115 has been opened to correct this.